edit/edit.js

```diff
@@ -508,7 +517,7 @@ window.onbeforeunload = () => {
   if (isCleanGlobal()) {
     return;
   }
-  updateLintReport(null, 0);
+  updateLintReportIfEnabled(null, 0);
   return confirm(t('styleChangesNotSaved'));
```

Comment:
js dialogs may be blocked by user setting, can you switch to `messageBox()` we use in manage.js for delete confirmation?